kuma/api/v1/views.py

```diff
@@ -382,3 +384,15 @@ def send_subscriptions_feedback(request):
         CATEGORY_MONTHLY_PAYMENTS, ACTION_SUBSCRIPTION_FEEDBACK, data["feedback"]
     )
     return HttpResponse(status=204)
+
+
+@api_view(["POST"])
+def create_subscription(request):
+    if not request.user.is_authenticated or not flag_is_active(request, "subscription"):
+        return Response(None, status=status.HTTP_403_FORBIDDEN,)
+
+    user = request.user
+    create_stripe_customer_and_subscription_for_user(
+        user, user.email, request.data.get("stripe_token", "")
```

Comment:
It's kinda weird to allow an empty string to be passed into `create_stripe_customer_and_subscription_for_user`
The errors you'd get in Sentry would be a lot easier to comprehend if we could avoid Stripe entirely. 
This line could become:
```suggestion
        user, user.email, request.data["stripe_token"]
```
so you'd get a `KeyError` instead of some strange Stripe API Error when the stripe_token is omitted in the POST read.

Reply:
That makes a whole lot of sense